packages/server/postgres/queries/generated/getLatestIntegrationSearchQueriesWithProviderIdQuery.ts

```diff
@@ -0,0 +1,52 @@
+/** Types generated for queries found in "packages/server/postgres/queries/src/getLatestIntegrationSearchQueriesWithProviderIdQuery.sql" */
+import { PreparedQuery } from '@pgtyped/query';
+
+export type IntegrationProviderServiceEnum = 'azureDevOps' | 'gitlab' | 'jiraServer' | 'mattermost' | 'msTeams';
+
+export type Json = null | boolean | number | string | Json[] | { [key: string]: Json };
+
+/** 'GetLatestIntegrationSearchQueriesWithProviderIdQuery' parameters type */
+export interface IGetLatestIntegrationSearchQueriesWithProviderIdQueryParams {
+  teamId: string | null | void;
+  userId: string | null | void;
+  service: IntegrationProviderServiceEnum | null | void;
+  providerId: number | null | void;
+}
+
+/** 'GetLatestIntegrationSearchQueriesWithProviderIdQuery' return type */
+export interface IGetLatestIntegrationSearchQueriesWithProviderIdQueryResult {
+  id: number;
+  userId: string;
+  teamId: string;
+  providerId: number | null;
```

Comment:
-2 The type is correctly generated here, just pass `null` as value if you want to query for `"providerId" is null`